db/schema.rb

```diff
@@ -783,6 +783,7 @@
     t.string "email"
     t.string "github_username"
     t.datetime "last_article_at", default: "2017-01-01 05:00:00"
+    t.datetime "latest_article_updated_at"
```

Comment:
This is just reordered, probably due to the recent switch from fix-db-schema-conflicts to strong-migrations.